doc/Geometry.rst

```diff
@@ -7,7 +7,7 @@ n_steps
 -------
 The network views each speech sample as a sequence of time-slices :math:`x^{(i)}_t` of
 length :math:`T^{(i)}`. As the speech samples vary in length, we know that :math:`T^{(i)}`
-need not equal :math:`T^{(j)}` for :math:`i \ne j`. For each batch, BRNN in TensorFlow needs
+need not equal :math:`T^{(j)}` for :math:`i \ne j`. For each batch, RNN in TensorFlow needs
 to know ``n_steps`` which is the maximum :math:`T^{(i)}` for the batch.
```

Comment:
We should remove this, we use dynamic RNNs in the training graph with no need for a fixed or known `n_steps` dimension.

Reply:
OK